### ISSUE TITLE
[enchancement] Suggestions from Samathingamajig

### ISSUE THREAD

**Issue by Samathingamajig** [jmslocum16/c2c-competition-guide#7]
### Original
![red, then black](https://snipboard.io/iW6Alp.jpg)
### Converted
![black, then](https://snipboard.io/bRnJog.jpg)
---
The reason for this suggestion is because the black piece goes first on game 1, and black being the second option causes confusion. This should be a simple change, just changing the order in the html document, making sure to change both the `<label>` and the `<input>`

---
### Original
<img src="https://user-images.githubusercontent.com/29130894/89497255-980d8f80-d781-11ea-8240-f820db7ff9c8.png" alt="alt text" width="350" height="300">

### Converted
<img src="https://user-images.githubusercontent.com/29130894/89497796-ad36ee00-d782-11ea-902f-1ce03d05f2bc.png" alt="alt text" width="350" height="300">

I have analyzed [the javascript displaying code](http://18.222.83.251:5000/static/js/watch.js), and understand it. If you would like me to add the functionality for this, I'd be happy to :)

---
### Original
![image](https://user-images.githubusercontent.com/29130894/89498756-7d88e580-d784-11ea-9fc9-1dfa5cb907a6.png)
![image](https://user-images.githubusercontent.com/29130894/89498740-7366e700-d784-11ea-8535-2579d320bb98.png)
### Converted
![image](https://user-images.githubusercontent.com/29130894/89498904-bcb73680-d784-11ea-9f14-c47b62211423.png)
![image](https://user-images.githubusercontent.com/29130894/89498891-b6c15580-d784-11ea-959d-5e580d8d9269.png)

This just lets someone click in the top left to go to /start/

**Comment by jmslocum16** [comment]
Great suggestions!
Unfortunately the github repo for the contest webapp also has all of the [c2c] bot code in there, so i don't want to share access that until after the competition :( but i'm happy to implement the circling change

**Comment by jmslocum16** [comment]
Well actually if you wanted to give me an updated watch.js file i can test it out and add it to the webapp

**Comment by Samathingamajig** [comment]
> Well actually if you wanted to give me an updated watch.js file i can test it out and add it to the webapp

Here it is: https://github.com/Samathingamajig/c2c-watch.js-update

I made a micro-git, the first commit is just the original code, the second is my updates

**Comment by Samathingamajig** [comment]
@jmslocum16 There is a big issue with what I changed, I had thought that lastMove meant the winning move, so now every move it'll check for a win (resource intensive) and not highlight the current drop, unless it coincidentally was the winning move. I'm fixing it whenever I get home, it should be good by 9:15 CT

# EDIT: Fixed issue

**Comment by jmslocum16** [comment]
Hey sam, for adding it!
I updated the site with your code and made a couple small fixes, you should see it live now!

**Comment by Samathingamajig** [comment]
> Hey sam, for adding it!
> I updated the site with your code and made a couple small fixes, you should see it live now!

The watch.js that you got from me wasn't the most updated version, commit # 3, which fixed the issue that you fixed also, but the main problem with commit # 2 was checking for winning pieces every turn, and of course, not highlighting the current piece unless it was a winning one.

Also, the submit page has a "<" after the phrase "Submit A Program" in the <title>

**Comment by Samathingamajig** [comment]
(also someone can submit a program with the same team-name and program-name and it overrides the current stored program)

**Comment by jmslocum16** [comment]
Yeah the overriding was desired - if you want to update your program you can.
Ah I see, my bad

**Comment by Samathingamajig** [comment]
> Yeah the overriding was desired - if you want to update your program you can.

Makes sense, overriding should be disabled on competition day so that someone can't replace someone else's bot. My team has a special file-naming scheme, so we hadn't tried replacing an existing program, and now the selection is filled with a whole lot of our bots. We have all of them stored locally, so if you want you can delete all of our [day 3]'s and [day 4]'s, except [day 3]-5-3 and [day 4]-3-1 to minimize the amount of programs, or you can keep them to show progress idk

Also, any updates on the questions for Hannah? I don't see any email from you about them.

![image](https://user-images.githubusercontent.com/29130894/89715692-0e221a00-d96d-11ea-91c7-6930d6a880ab.png)

**Comment by jmslocum16** [comment]
Yeah that's a fair point. Yeah I'm fine keeping them. I guess it would have been good to be explicit about the expected submission behavior either way.
And I have not heard back from Hannah, last I heard she would get back to me, though I would expect that with monday being the last day, that there won't end up being a slack workspace. And currently only 3 teams have submitted a working bot, so I would guess by now that the other couple teams won't be submitting anything.
